src/model.rs

```diff
@@ -387,7 +392,7 @@ pub struct Group {
 	pub channel_id: ChannelId,
 	pub icon: Option<String>,
 	pub last_message_id: Option<MessageId>,
-	pub last_pin_timestamp: Option<String>,
+	pub last_pin_timestamp: Option<DateTime<FixedOffset>>,
```

Comment:
Chrono documentation shows that it does operate on strings in RFC3339 format.  NaiveDateTime implementation outputs a DateTime without the offset, https://docs.rs/chrono/0.3.1/src/chrono/naive/datetime.rs.html#1321-1344

DateTime combines the string representation of NaiveDateTime with the string representation of the TimeZone

https://docs.rs/chrono/0.3.1/src/chrono/offset/fixed.rs.html#121-133

This means it will parse and return strings in the correct format.